### PR TITLE
Send back kyc_max_attempts in serializedKYC

### PR DESCRIPTION
**Pull request by NullSoldier**
## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: src/jumio-kyc/kyc.controller.spec.ts [hunk 1]
```diff
@@ -39,6 +41,7 @@ describe('KycController', () => {
   beforeAll(async () => {
     app = await bootstrapTestApp();
     prisma = app.get(PrismaService);
+    config = app.get(ConfigService);
```
Comment (rohanjadvani) [inline src/jumio-kyc/kyc.controller.spec.ts:44]:
```suggestion
    config = app.get(ApiConfigService);
```

Comment (NullSoldier) [reply src/jumio-kyc/kyc.controller.spec.ts]: Fixed

File: src/jumio-kyc/utils/serialize-kyc.ts [hunk 2]
```diff
@@ -12,12 +13,17 @@ export function serializeKyc(
   canAttemptReason: string,
   canCreate: boolean,
   canCreateReason: string,
+  config: ApiConfigService,
 ): SerializedKyc {
   assert.ok(redemption.jumio_account_id);
 
+  const maxAttempts =
```
Comment (rohanjadvani) [inline src/jumio-kyc/utils/serialize-kyc.ts:20]: Feels a little weird to pass an injected dependency into a serializer. I think it'd be cleaner if we did this in the caller (or just passing the config value as a default)

**Review by rohanjadvani — APPROVED** [review]

File: src/jumio-kyc/utils/serialize-kyc.ts (hunk 2 above)
Comment (NullSoldier) [reply src/jumio-kyc/utils/serialize-kyc.ts:20]: That is a better way but it it requires every test to change that tests this to do that

**Pull request merged**